Assets/ScriptsWithDependencies/RegistrationManager.cs

```diff
@@ -91,7 +101,28 @@ public void VerifyUsernameOnValueChanged(string usernameInput)
         }
         else
         {
-            InfoText.text = "Benutzername muss zwischen 5-20 Zeichen lang sein";
+            InfoText.text = "Benutzername muss zwischen 3-20 Zeichen lang sein";
+            SubmitButton.interactable = false;
+        }
+    }
+
+    /// <summary>
+    /// called after user tipped in the first password field.
+    /// If the given password is valid, it will be compared to the second one
+    /// which confirms whether the user entered the same password twice
+    /// </summary>
+    /// <param name="passwordInput">first input field for password</param>
+    public void CheckPasswordsAreSameAfterTippedOnFirstPassField(string passwordInput)

```

Comment:
Ist das ein Typo? Was ist tipped? Tipping im englischen ist eine "Donation" meineswissens.
```suggestion
    public void CheckPasswordsAreSameAfterTypingOnFirstPassField(string passwordInput)
```
Vielleicht so?

Reply:
stimmt, erledigt